src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx

```diff
@@ -122,111 +122,113 @@
 Copyright (C) Microsoft Corporation. All rights reserved.</value>
   </data>
   <data name="ShellHelp" xml:space="preserve">
-    <value>PowerShell[.exe] [-Version] [-NoLogo] [-NoExit]
-    [-NoProfile] [-NonInteractive] [-Interactive]
-    [-InputFormat {Text | XML}] [-OutputFormat {Text | XML}]
-    [-WindowStyle &lt;style&gt;] [-EncodedCommand &lt;Base64EncodedCommand&gt;]
-    [-ConfigurationName &lt;string&gt;]
-    [-Command { - | &lt;script-block&gt; [-args &lt;arg-array&gt;]
-                  | &lt;string&gt; [&lt;CommandParameters&gt;] } ]
-    [-File &lt;filePath&gt; &lt;args&gt;] [-ExecutionPolicy &lt;ExecutionPolicy&gt;]
```

Comment:
I don't think the syntax block should be removed.

Reply:
I agree, we should leave this

Reply:
The idea was to follow Bash style. Is it preffered?
You suggest Git style.
Final conclusion?

Reply:
In this case, I prefer Git style seeing the full syntax.